Dockerfile

```
@@ -112,6 +112,21 @@ COPY --from=builder /app/_build/prod/rel/wanderer_notifier ./
 COPY scripts/start_with_db.sh scripts/db_operations.sh /app/bin/
 RUN chmod +x /app/bin/start_with_db.sh /app/bin/db_operations.sh
 
+# Add a startup validation script to ensure CONFIG_PATH isn't duplicated
+RUN echo '#!/bin/bash\n\
```

Hadolint (2.12.0)

[info] echo may not expand escape sequences. Use printf.

(SC2028)